sys-libs/lwp/lwp-2.8.ebuild

```diff
@@ -1,8 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit ltprune
+EAPI=7
```

Comment:
Hrm, I'd kinda want this to be EAPI-8 given it'd allow to remove `--disable-static` and drop src_configure -- but, given this is straight-to-stable and still kinda early, I'll pass on it. Still wanted to mention it though.